client/src/containers/project/overview/component.tsx

```diff
@@ -151,6 +152,17 @@ const ProjectOverview = () => {
           <p className="font-semibold uppercase text-grey-20">About</p>
           <p className="text-xl">{description}</p>
         </div>
+
+        <div className="space-y-3">
+          <p className="font-semibold uppercase text-grey-20">Data Source</p>
+          <p className="text-xl">
+            FORA members have provided the project information displayed on this page. Please{' '}
+            <Link href="/privacy-policy">
```

Comment:
Next's Link will render a `<a>` automatically, so there's no need to set it. If you need to add `classnames` you can do it in the very `<Link>` component and it will be applied to the `<a>`: https://nextjs.org/docs/app/api-reference/components/link#props